hack/verify-govet.sh

```diff
@@ -21,8 +21,7 @@ FAILURE=false
 test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
 for test_dir in $test_dirs
 do
-  go tool vet -shadow=false \
-              $test_dir
+  GOPATH=${OS_ROOT}/_output/local/go:${OS_ROOT}/Godeps/_workspace:$GOPATH go tool vet -shadow=false $test_dir
```

Comment:
I think we should remove the trailing `:$GOPATH` since if something's not copied into `OS_ROOT` we want to fail fast, not use random stuff from the host's environment.